fe1-web/src/features/rollCall/screens/RollCallOpened.tsx

```diff
@@ -71,18 +71,13 @@ const RollCallOpened = () => {
     addOwnToken().catch((e) => console.error(e));
   }, [lao, rollCallID, toast]);
 
-  const handleError = (err: any) => {
+  const handleError = (err: string) => {
     console.error(err);
-    // The "err" object might be an exception, take the message property if it exists
-    toast.show(
-      err?.message ||
-        (typeof err === 'string' ? err : 'Unkown error, please check the console and report it!'),
-      {
-        type: 'danger',
-        placement: 'top',
-        duration: FOUR_SECONDS,
-      },
-    );
+    toast.show(err, {
+      type: 'danger',
+      placement: 'top',
+      duration: FOUR_SECONDS,
+    });
```

Comment:
Why is this changed back to the old way of handling errors? Did I miss something?

Reply:
Nope, that must be a bad conflict fixing thing I did